modules/ztp-preparing-factory-env.adoc

```diff
@@ -0,0 +1,23 @@
+// CNF-1500 ZTP - preparing to install ACM
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+[id="ztp-acm-preparing-to-install-disconnected-acm_{context}"]
+= Preparing the factory environment
+
+Before you can provision clusters at scale in the factory setting, you must install the {product-title} and OpenShift Data Foundation on the designated hub cluster. 
```

Comment:
Don't we also require RHACM on the hub/factory cluster as well?

Reply:
@jparrill Don't we also require RHACM on the hub/factory cluster as well. I think the deploy pipeline takes care of installing RHACM on hub

Reply:
The deploy pipeline takes care of installing ACM. We require a bare, plain, OCP cluster that we can install everything else on.